autoscheduler/frontend/src/tests/ui/SchedulingPage.test.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import {
+  render, fireEvent, waitForDomChange, queryByText as queryContainerByText,
+} from '@testing-library/react';
+import autoSchedulerReducer from '../../redux/reducer';
+import SchedulingPage from '../../components/SchedulingPage/SchedulingPage';
+
+describe('Scheduling Page UI', () => {
+  describe('shows an error message', () => {
```

Comment:
```suggestion
  describe('indicates that there are no schedules', () => {
```
Something like this, the way it's worded now confused me at first since it isn't really an error message, just placeholder content